x-pack/plugins/triggers_actions_ui/public/application/context/actions_connectors_context.tsx

```diff
@@ -5,15 +5,19 @@
  */
 
 import React, { createContext, useContext } from 'react';
-import { ActionType } from '../../types';
+import { HttpSetup, ToastsApi, ApplicationStart } from 'kibana/public';
+import { ActionTypeModel } from '../../types';
+import { TypeRegistry } from '../type_registry';
 
 export interface ActionsConnectorsContextValue {
-  addFlyoutVisible: boolean;
-  editFlyoutVisible: boolean;
-  setEditFlyoutVisibility: React.Dispatch<React.SetStateAction<boolean>>;
-  setAddFlyoutVisibility: React.Dispatch<React.SetStateAction<boolean>>;
-  actionTypesIndex: Record<string, ActionType> | undefined;
-  reloadConnectors: () => Promise<void>;
+  http: HttpSetup;
+  actionTypeRegistry: TypeRegistry<ActionTypeModel>;
+  toastNotifications?: Pick<
```

Comment:
It seems strange to make toast notifications optional? We're moving towards using this to display error messages during a save and there's a few places relying on it to indicate something went wrong.